tests/util.js

```diff
@@ -200,5 +203,86 @@ export default {
 				}
 			],
 		},
+		{
+			name: "isRegisteredProperty()",
+
+			beforeEach () {
+				let iframe = document.createElement("iframe");
+				document.body.append(iframe);
+
+				this.data.iframe = iframe;
+			},
+
+			run (...properties) {
+				return properties.map(property => isRegisteredProperty(`--${property.id}`, this.data.iframe.contentWindow));
+			},
+
+			afterEach () {
+				this.data.iframe.remove();
+			},
+
+			args: [types.any, { ...types.any, id: "any-non-inherited", inherits: false }, types.number],
+			expect: [false, true, true],
+			tests: [
+				{
+					name: "Inline style",
+					beforeEach () {
+						this.parent.beforeEach.call(this);
+						this.data.iframe.contentDocument.head.insertAdjacentHTML(
+							"beforeend",
+							`<style>${this.args.map(property => `
+								@property --${property.id} {
+									syntax: "${property.syntax}";
+									inherits: ${property.inherits ?? "true"};
+									initial-value: ${property.initialValue};
+								}`)
+								.join("\n")}
+							</style>`,
+						);
+					},
+				},
+				{
+					name: "Adopted stylesheet",
+					beforeEach () {
+						this.parent.beforeEach.call(this);
+						adoptCss(`${this.args.map(property => `
+							@property --${property.id} {
+								syntax: "${property.syntax}";
+								inherits: ${property.inherits ?? "true"};
+								initial-value: ${property.initialValue};
+							}`)
+							.join("\n")
+						}`, this.data.iframe.contentDocument);
+					},
+				},
+				{
+					name: "External stylesheet",
+					async beforeEach () {
+						this.parent.beforeEach.call(this);
+						let link = Object.assign(document.createElement("link"), {
+							rel: "stylesheet",
+							href: "./util/properties.css",
+						});
+
+						this.data.iframe.contentDocument.head.append(link);
```

Comment:
This is fine, but this is a bit shorter / more readable:

```suggestion
						this.data.iframe.contentDocument.head.insertAdjacentHTML("beforeend", '<link rel="stylesheet" href="util/properties.css" />');
```